mx.py

```diff
@@ -9856,7 +9856,7 @@ def log(msg=None):
     if msg is None:
         print
     else:
-        print msg
```

Comment:
oh wow, that's really unfortunate. I guess the print *function* does not suffer from this problem?